packages/components/src/components/domainproperties/DomainRow.tsx

```diff
@@ -67,14 +69,14 @@ interface IDomainRowProps {
     fieldError?: DomainFieldError
     onDelete: (any) => void
     onExpand: (index?: number) => void
-    isDragDisabled: boolean
     showDefaultValueSettings: boolean
     defaultDefaultValueType: string
     defaultValueOptions: List<string>
     appPropertiesOnly?: boolean
```

Comment:
I don't think you have to do it as part of this PR, but we should add a TODO next to these appPropertiesOnly to convert them into more options in the IDomainFormDisplayOptions interface